Added Mobile menu

PrangonGhose opened the pull request:
Added mobile menu according to the figma template and functionalities to the website.

BenMukebo commented on style.css:
```diff
@@ -119,6 +123,53 @@
+    width: 100%;
+  }
+
+  .mobile-menu-listItems {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 70px;
+    margin-bottom: 0;
+    list-style: none;
+    position: relative;
+  }
```
- [OPTIONAL] Great job so far :raised_hands: but I think it would be better if you could use a `position: fixed;`  instead of absolute. Fixed position prevents your navigation page menu page from scrolling. The navigation is always fixed to the page instead of scrolling. Please change this. This will make your website similar to the original design. 💯 🥇 


- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 


- [ ]  The navigation menu page has to follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). Kindly make sure to use the same font-size: 32px; line-height: 44px; ...🎨 👨🏻‍🎨 





-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
| ![tresbad](https://user-images.githubusercontent.com/79058364/174133622-11c7aff6-13ab-427e-ad60-75136954d497.png)| ![raw githack com_PrangonGhose_My-Portfolio_Mobile-menu_index html](https://user-images.githubusercontent.com/79058364/201791665-5e9066f8-e0f0-489c-9ff9-b0abdcc3971e.png)|

BenMukebo reviewed (changes requested):
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository have been set up correctly. ✔️  
- Good commit history ✔️ 
-  When the user clicks the hamburger button, the mobile menu appears.✔️ 
- When the user clicks the close (X) button, the mobile menu disappears.✔️ 
- When the user clicks any of the mobile menu options, a correct part is displayed and the mobile menu disappears 👏🏻 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Great job: adding the README file 👏🏻 Let's make it more descriptive, Kindly, it would be nice to add explicit steps for cloning and running your app, as this will help you, non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd..`, etc. ✍🏻 

- [ ] This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 👨🏻‍🎨 

- [ ] Please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮 

- [ ] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL]  Please let's make sure that contributions to a group project are distributed equally among the members (commit messages should be fairly equal 👨🏻‍🎨 

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PrangonGhose commented:
***Changes made after first review***
- Edited readme file as specified
- Fixed issues with mobile menu scrolling
- Fixed issues with background scrolling

ichala reviewed (approved):
Hi @PrangonGhose  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966](https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif)

# To Highlight 💪🏻

-  Changes Implemenetd✔️ 



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
